src/utils/DataManipulation.js

```diff
@@ -0,0 +1,62 @@
+import { AsyncStorage } from 'react-native';
+import { filter } from 'lodash';
+import moment from 'moment';
+
+const QUALITIES = ['excellent', 'verygood', 'good', 'okay'];
+
+class DataManipulation {
+    formatData(data, qualities = QUALITIES) {
```

Comment:
Really this is formatting the days -- `formatDays(days, qualities...)` is much easier to understand imo than `formatData`

Reply:
Agreed! Resolved with 82d55bb6bd8b6311bb6f2b9ac73a5b18df423827